packages/gmx/src/contract-addresses.ts

```diff
@@ -0,0 +1,14 @@
+export const GMX_ROUTERV1_ADDRESS = '0xabbc5f99639c9b6bcb58544ddf04efa6802f4064'
+export const GMX_ROUTERV2_ADDRESS = '0x3B070aA6847bd0fB56eFAdB351f49BBb7619dbc2'
+export const DEFAULT_TOKEN_LIST_URL = [
```

Comment:
should be able to do `const DEFAULT_TOKEN_LIST_URL: Address[] = [ ... ]` and you shouldn't have to do the type casting other places 

`Address` comes from viem